src/main/java/org/isf/stats/rest/ReportsController.java

```diff
@@ -21,13 +21,9 @@
  */
 package org.isf.stats.rest;
 
```

Comment:
Please use the  IDE formatting files for in the `.ide-settings` directory.  The changes to the imports here do not match project standards:

1. static imports
2. java
3. javax
4. org
5. com
6. everything else

Thanks

Reply:
Done!